src/components/donation/Steps/Submit/Crypto/Checkout.tsx

```diff
@@ -0,0 +1,113 @@
+import { Info, LoadingStatus } from "components/Status";
+import { chains } from "constants/chains";
+import { isDisconnected, useWalletContext } from "contexts/WalletContext";
+import { maskAddress } from "helpers";
+import { PropsWithChildren } from "react";
+import { CryptoSubmitStep } from "slices/donation";
+import { ConnectedWallet } from "types/wallet";
+import TxSubmit from "./TxSubmit";
+import WalletSelection from "./WalletSelection";
+
+type Props = CryptoSubmitStep & { classes?: string };
+
+export default function Checkout({ classes = "", ...props }: Props) {
+  const wallet = useWalletContext();
+  const chainID = props.details.chainId.value;
+
+  if (wallet === "loading") {
+    return (
+      <Container classes={classes} donation={props}>
+        <LoadingStatus classes="py-2 mb-6 text-sm text-gray-d1">
+          Connecting wallet..
+        </LoadingStatus>
+      </Container>
+    );
+  }
+
+  if (isDisconnected(wallet)) {
+    return (
+      <Container classes={classes} donation={props}>
+        <WalletSelection chainID={chainID} wallets={wallet} classes="mt-2" />
+      </Container>
+    );
+  }
+
+  if (!wallet.supportedChains.includes(chainID)) {
+    return (
+      <Container classes={classes} donation={props}>
+        <Info classes="mt-2">Connected wallet doesn't support this chain.</Info>
+        <button
+          className="btn-outline-filled px-4 py-2 mt-2 text-xs font-normal font-work justify-self-center"
+          type="button"
+          onClick={wallet.disconnect}
+        >
+          change wallet
+        </button>
+      </Container>
+    );
+  }
+
+  if (chainID !== wallet.chainId) {
+    return (
+      <Container classes={classes} donation={props}>
+        {wallet.switchChain ? (
+          <>
+            <Info classes="mt-2">
+              Your wallet is not connected to your selected chain.
+            </Info>
+            <button
+              disabled={wallet.isSwitching}
+              type="button"
+              onClick={() => wallet.switchChain?.(chainID)}
+              className="btn-outline-filled px-4 py-2 mt-2 text-xs font-normal font-work"
+            >
+              Switch to {chains[chainID].name}
+            </button>
+          </>
+        ) : (
+          <Info classes="mt-2">
+            Kindly set your wallet network to your selected chain.
+          </Info>
+        )}
+      </Container>
+    );
+  }
+
+  return (
+    <Container donation={props} wallet={wallet} classes={classes}>
+      <div className="flex items-center gap-2 mt-2">
+        <div className="size-2 rounded-full bg-green drop-shadow-[0_0_3px_rgba(126,198,130,0.9)]" />
+        <Image src={wallet.logo} className="size-5 rounded-full" />
+        <span className="text-sm">{maskAddress(wallet.address)}</span>
+        <button
+          type="button"
+          onClick={wallet.disconnect}
+          className="bg-gray-l5 hover:bg-gray-l4 text-xs px-2 py-1 rounded"
```

Comment:
@SovereignAndrey 
> We should make the change wallet button match the other button styles:

> rounded corners
have some padding around text label
use title-case on label text (not uppercase)

Applied requested changes
<img width="501" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/c34897e2-0a0e-40a1-88ef-42b10d04e5ae">

Reply:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/85138450/aa9c93ad-99a2-4204-b7e6-d25dfa5adc0a)
@ap-justin Change the button's class from `rounded` >> `rounded-full` I think it's OK.

Reply:
updated to rounded full https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2701/commits/1a3da1d4375c789c8071a4cdd2f54fde75f369b8
<img width="634" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/923a7b09-ebc0-4b3a-ac14-2246fadf9d17">